_posts/2022-02-18-ols-is-hiring.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Job vacancy: Community Researcher and Programme Coordinator - OLS is hiring!"
+authors:
+- bebatut
+- emmyft
+- malvikasharan
+- yochannah
+image: ../images/2022-02-18-for-hire.jpg
+photos:
+  name: Photo by Clem Onojeghuo on Unsplash
+  url: https://unsplash.com/@clemono
+---
+
+OLS is hiring a **Community Researcher and Programme Coordinator**.
+
+_Note: If you like the scope of this job but unsure if your profile fits this role - please apply anyway. We are very happy to be contacted for general queries, please email [jobs@openlifesci.org](mailto:jobs@openlifesci.org). You can also book a call with the Executive Director, Yo Yehudi, for specific questions about this vacancy: [book a 15 minute chat](https://calendly.com/yo-yehudi/ols-chat-with-yo?month=2022-02)._
+
+Open Life Science (OLS) is hiring an open, empathic, and creative colleague to coordinate our cohort-based training and mentoring programmes and lead a research-based study to understand the long-term impacts of our work.
+
+Open Life Science (OLS, [{{ site.url }}]({{ site.url }})) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. OLS is committed to delivering high-quality training, mentoring and peer-based support to our community members.
+
+## Position purpose
+
+This role encompasses two areas of OLS' work: community research and programme coordination. As a community researcher, you will help to carry out peer-reviewed research to understand transformational effects of open research training and mentoring interventions. As a programme coordinator, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. 
```

Comment:
```suggestion
This role encompasses two areas of OLS' work: community management and community research. As a community manager, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information openly. As a community researcher, you will help to carry out peer-reviewed research to understand the transformational effects of open research training and mentoring interventions.

Reply:
@malvikasharan I had to do some manual conflict resolutions between the two sets of edits - can you check if this is the final language you wanted?